CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## 6.8.0 (Unreleased)
 
+## 5.44.2 (October 14, 2024)
```

Comment:
Do we want to keep this 5.44.2 changelog on top of the 6.xx.x? Or should we leave it on top of 5.44.1?

Reply:
I personally prefer chronological order, although I see `5.44.1` was different! I'm gonna split out the 4.X and 5.X changelogs soon anyways.

Reply:
LGTM